### PR TITLE
feat: add dark mode for callback page

### PR DESCRIPTION
**Pull request by bayandin**
Successful `neonctl auth` redirects you to a really bright callback page.
This PR adds a dark theme to make the page friendly for owls.

Old Light mode:
old light mode

New Light mode:
new light mode

New Dark mode:
new dark mode

### CODE REVIEW

File: src/callback.html [hunk 1]
```diff
@@ -15,6 +15,12 @@
   justify-content: center;
   align-items: center;
 }
+@media (prefers-color-scheme: dark) {
+  body, html {
+    background-color: #131313;
+    color: #ffffff;
```
Comment (bayandin) [inline src/callback.html:21]: I've taken some colours from the neon console, please let me know if I need to adjust them to match the Neon style.

Comment (PlayLikeNeverB4) [reply src/callback.html:21]: On the console we use
```
background-color: #191919;
color: #bfbfbf;
```

**Review by PlayLikeNeverB4 — APPROVED** [review]

File: src/callback.html (hunk 1 above)
Comment (bayandin) [reply src/callback.html]: A-ha, thanks!

While we're here, do we need to update the light theme (currently we use just black and white)?

Comment (PlayLikeNeverB4) [reply src/callback.html]: For the light theme it's
```
background-color: #ffffff;
color: #2d374c;
```

**Pull request merged**